src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,28 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+
```

Comment:
```suggestion
public class CandidateValidator implements Predicate<Candidate> {
    private static final int OPTIMAL_AGE = 35;
    private static final int VALID_YEARS_IN_UKR = 10;
    private static final String VALID_NATIONALITY = "Ukrainian";
```

I meant only `private static final boolean ALLOWED = true;` constant, we need the rest